src/search.cpp

```diff
@@ -76,8 +76,7 @@ namespace {
   }
 
   constexpr int futility_move_count(bool improving, Depth depth) {
-    return improving ? (3 + depth * depth)
-                     : (3 + depth * depth) / 2;
+    return (3 + depth * depth) >> (improving ? 0 : 1);
```

Comment:
I find the ternary for boolean’s pretty ugly I would suggest to simply use !improving though you might have to rerun this test

Reply:
Okay then I think it’s fluke, like vondele mentioned.